### DISCUSSION TITLE
Preferred orientation (45 degrees angle)

### DISCUSSION THREAD

**Discussion by rvbcrs** [OrcaSlicer/OrcaSlicer#455] [General]
Hi, first of all great work, I love SoftFever so far!
Maybe I just can't find it, but in SuperSlicer there is a setting to set the preferred orientation to 45 degrees angle. Is there a way to do this in SoftFever? or are there plans to add a feature like this?

Cheers

**Comment by erlis** [comment]
+1 to this comment! Is there a way to do this?

**Comment by ToXIc69** [comment]
+1 to this comment! please add feature.

**Comment by janusr** [comment]
This setting does the trick - stl only though.
 
![image](https://github.com/SoftFever/OrcaSlicer/assets/8723971/67c0527e-e567-4227-8d32-a8cc9edbe20c)

**Reply by ToXIc69** [reply]
ahh there it is! thanks!
